CrossDeviceRockPaperScissorsKotlin/build.gradle

```
@@ -1,13 +1,14 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
+    ext.kotlin_version = '1.7.10'
     repositories {
         mavenCentral()
     }
 }
 plugins {
-    id 'com.android.application' version '7.1.3' apply false
-    id 'com.android.library' version '7.1.3' apply false
-    id 'org.jetbrains.kotlin.android' version '1.6.21' apply false
+    id 'com.android.application' version '7.2.1' apply false
+    id 'com.android.library' version '7.2.1' apply false
+    id 'org.jetbrains.kotlin.android' version '1.7.10' apply false
```

Thank you for the contribution!
See below a suggested change.
```suggestion
    id 'org.jetbrains.kotlin.android' version "$kotlin_version" apply false
```